README.md

```diff
@@ -7,64 +7,78 @@
        ########: ##:::: ##: ##:::::::: ##::::::::'####: ##::::::: ########:. ######::::: ##::::
       ........::..:::::..::..:::::::::..:::::::::....::..::::::::........:::......::::::..:::::
 
-
 Cli app to generate and publish Zapp plugin manifest.
 
 #### Installation
 
 Install via Homebrew
+
 ```bash
 brew tap applicaster/tap
 brew install zappifest
 ```
 
 #### Upgrade
+
 ```bash
 brew upgrade zappifest
 ```
 
 ## Usage
+
 ### Init
+
 Zappifest allows fast configuration for Zapp plugin-manifest.json file.
 Just run `zappifest init` and follow the instructions.
 
 ### Publish
 
 #### Prerequisites
+
 Reach Applicaster support team to generate User access-token.
 
 The tool allow you to publish the plugin to Zapp.
 
-Run `zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token>`
+Run `zappifest publish --manifest <path-to-manifest-json-file> --access-token <zapp-access-token> --account <account-id>`
```

Comment:
We can actually remove the --manifest flag, you can just run publish <path_to_manifest>

Reply:
@ranmeirman i prefer to do it in separate PR, not related to this PR